buildSrc/src/main/kotlin/MavenPomUtils.kt

```diff
@@ -0,0 +1,47 @@
+@file:JvmName("MavenPomConfiguration")
+
+import org.gradle.api.publish.maven.*
```

Comment:
```suggestion
import org.gradle.api.publish.maven.*
```

Since all the build scripts are Kotlin now, the JvmName could be removed?

Reply:
Applied